mslib/mscolab/models.py

```diff
@@ -31,6 +31,21 @@
 from passlib.apps import custom_app_context as pwd_context
 from mslib.mscolab.app import db
 from mslib.mscolab.message_type import MessageType
+import sqlalchemy.types as types
+
+
+class Awaredatetime(types.TypeDecorator):
```

Comment:
```suggestion
class AwareDateTime(types.TypeDecorator):
```

The class name should stay consistent with the usual CamelCase naming convention for classes.